### ISSUE TITLE
Server sent bad data message (for radio stream URL that works in VLC)

### ISSUE THREAD

**Issue by Durhammer3111** [clementine-player/Clementine#6840]
### Before posting

Please follow the steps below and check the boxes with [x] once you did the step.

- [ x] I checked the issue tracker for similar issues
- [ x] I checked the [changelog](https://github.com/clementine-player/Clementine/blob/master/Changelog) if the issue is already resolved
- [x ] I tried the latest Clementine build from [here](https://builds.clementine-player.org/)

### System information

Please provide information about your system and the version of Clementine used.

- Operating System:MX Linux 19.3
- Clementine version: 1.3.1

### Expected behaviour / actual behaviour 
Play radio stream / get error message in popup window saying "Server sent bad data"
### Steps to reproduce the problem (only for bugs)
Simply try adding and playing a new radio stream with this URL: https://live.wostreaming.net/direct/capitolbroad-wclyamaac-ibc1
(It works fine in VLC but I don't like VLC's UI.)

[clementine.log](https://github.com/clementine-player/Clementine/files/5577847/clementine.log)

**Comment by jbroadus** [comment]
The URL works for me on the latest. I don't know how much MX deviates from Debian, but you could try to install the latest Buster image available here:
https://github.com/clementine-player/Clementine/releases/tag/1.4.0rc1-351-g1daf43f91

**Comment by Durhammer3111** [comment]
(Sorry if this got sent twice, hit reply and not "reply list.")

Nope, sorry! I DID also try MX's "Test" version with the same results. 
Installed the Buster image, and did get a bit more logging (log 
attached), but it doesn't appear to have helped. On the other hand, the 
stream parsing is MUCH better, as some stations' metadata now shows the 
artist in the artist column and not in the title, and the song and 
artist infos are often actually fetched and shown! Yay!

**Comment by jbroadus** [comment]
Looks like a similar issue on a different project: https://github.com/ebruck/radiotray-ng/issues/110

Do you see the issue with gst-play-1.0?

**Comment by Durhammer3111** [comment]
Yes, same result. I had seen that issue but hadn't tried the command. 
Also ignored it 'cause there didn't seem to be a workaround or fix for 
that one.

It's probably moot for me now. I had moved from Mint 19.3 to MX 19.3 and 
Cantata wasn't working (dumb user error -- thought mpd was installed but 
it wasn't) and Clementine was already installed so figured I'd try it 
and ran into this issue. As reported, VLC had no issue but the UI sucks. 
Now that I've got Cantata working again, this and another wostreaming 
server station is/are working fine in that, so I'll probably forget 
about Clementine again.

But if it's fixable, it oughta be fixed! Thanks!
